compiler/passes/ResolveScope.cpp

```diff
@@ -830,6 +831,30 @@ Symbol* ResolveScope::lookupNameLocally(const char* name, bool isUse) const {
   return retval;
 }
 
+Symbol* ResolveScope::lookupPublicImports(const char* name) const {
+  UseImportList useImportList = mUseImportList;
+  Symbol *retval = NULL;
+
+  for (size_t i = 0; i < useImportList.size(); i++) {
```

Comment:
Michael pointed out to me in my last PR that we can use the `for_vector_allowing_0s` macro in places like this